EddystoneBeacon.cpp

```diff
@@ -28,7 +29,8 @@ static const char* EDDYSTONE_URL_BEACON_SUFFIX_SUBSTITUTIONS[] = {
   ".net",
   ".info",
   ".biz",
-  ".gov"
+  ".gov",
+  ""
```

Comment:
This changes are not up to spec, see: https://github.com/google/eddystone/tree/master/eddystone-url#eddystone-url-http-url-encoding